src/core/border_router/routing_manager.cpp

```diff
@@ -628,16 +629,22 @@ void RoutingManager::SendRouterAdvertisement(RouterAdvTxMode aRaTxMode)
         SuccessOrExit(error = mRioAdvertiser.AppendRios(raMsg));
     }
 
+    if (Get<InfraIf>().GetLinkLayerAddress(&sllao) == kErrorNone) {
+        raMsg.AppendLinkLayerOption(&sllao, true);
```

Comment:
Maybe it's good to reconsider the use of `bool` here. When reading the code, it's not clear what a `true` input indicates. 

Looking at the code below, it seems to indicate whether it's the source or target address. 

Some suggestions:

* We seem to only care about the source address option, so could we remove the input altogether?
* We could consider defining an `enum Type` (in `LinkLayerOption` class) to indicate the type?